contracts/utils/LowLevelCallUtils.sol

```diff
@@ -67,4 +71,30 @@ library LowLevelCallUtils {
             revert(0, returndatasize())
         }
     }
+
+    /**
+     * @dev An optimised function to compute the sha3 of the lower-case
+     *      hexadecimal representation of an Ethereum address.
+     * @param addr The address to hash
+     * @return ret The SHA3 hash of the lower-case hexadecimal encoding of the
+     *         input address.
+     */
+    function sha3HexAddress(address addr) internal pure returns (bytes32 ret) {
+        assembly {
+            for {
+                let i := 40
+            } gt(i, 0) {
+
+            } {
+                i := sub(i, 1)
+                mstore8(i, byte(and(addr, 0xf), lookup))
+                addr := div(addr, 0x10)
+                i := sub(i, 1)
+                mstore8(i, byte(and(addr, 0xf), lookup))
+                addr := div(addr, 0x10)
+            }
+
+            ret := keccak256(0, 40)
+        }
+    }
```

Comment:
Don't we already have a copy of this somewhere that we could import?

Reply:
It only exists on `ReverseRegistrar` https://github.com/ensdomains/ens-contracts/blob/21736916300b26cb8ea1802dbf6c9ff054adaeab/contracts/reverseRegistrar/ReverseRegistrar.sol#L164 which I cannot import

I can refactor so that ReverseRegistrar can import the function from `LowLevelCallUtils` instead

Reply:
Yes please!